packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -77,13 +89,60 @@ abstract contract FundingRateApplier is Testable, Lockable {
      * @notice Constructs the FundingRateApplier contract. Called by child contracts.
      * @param _finderAddress Finder used to discover financial-product-related contracts.
      */
-    constructor(address _finderAddress) public {
-        finder = FinderInterface(_finderAddress);
+    constructor(
+        FixedPoint.Unsigned memory _fundingRateBondPercentage,
```

Comment:
It will be nice to read `_fundingRateBondPercentage` and `_fundingRateRewardRate` from the ConfigStore

Reply:
Yeah, 100% agreed.

Reply:
Merging this in soon.